frontend/.directory

```diff
@@ -0,0 +1,6 @@
+[Dolphin]
+Timestamp=2020,4,6,12,21,29
+Version=4
+
+[Settings]
+HiddenFilesShown=true
```

Comment:
Is this something that should be in .gitignore?

Reply:
Yep, don't know how these got included, I'll fix asap

Reply:
Removed